packages/host/tests/integration/search-index-test.gts

```diff
@@ -976,6 +976,65 @@ module(`Integration | search-index`, function (hooks) {
     }
   });
 
+  test('can capture css when indexing a card', async function (assert) {
+    let { realm } = await setupIntegrationTestRealm({
+      loader,
+      contents: {
+        // we use a string src here because shimmed modules are skipped by the indexer
+        'person.gts': `
+          import { contains, field, Component, CardDef } from 'https://cardstack.com/base/card-api';
+          import StringField from 'https://cardstack.com/base/string';
+
+          export class Person extends CardDef {
+            @field firstName = contains(StringField);
+            static isolated = class Isolated extends Component<typeof this> {
+              <template>
+                <h2><@fields.firstName /></h2>
+                <style>
+                  h2 {
+                    color: red;
+                  }
+                </style>
+              </template>
+            };
+          }
+        `,
+        'mango.json': {
+          data: {
+            attributes: { firstName: 'Mango' },
+            meta: {
+              adoptsFrom: { module: `./person`, name: 'Person' },
+            },
+          },
+        },
+      },
+    });
+
+    let indexer = realm.searchIndex;
+    let scope = `data-scopedcss-a61576b4d4-6ad446b263`;
```

Comment:
This is surely fine for now but we could extract [the composition of this attribute](https://github.com/cardstack/glimmer-scoped-css/blob/main/glimmer-scoped-css/src/ast-transform.ts#L24) if it ends up changing.